src/components/pages/SideDrawer/SideDrawer.js

```diff
@@ -104,11 +105,25 @@ const useStyles = makeStyles(theme => ({
 export default function SideDrawer(props) {
   const classes = useStyles();
   const theme = useTheme();
-  const [open, setOpen] = React.useState(true);
+  const [open, setOpen] = useState(true);
+  const [mobile, setMobile] = useState(window.innerWidth < 769);
   const { logout } = props;
 
   const admin = window.localStorage.getItem('Admin');
 
+  useEffect(() => {
+    window.addEventListener(
+      'resize',
+      () => {
+        const mobile = window.innerWidth < 769;
+        if (mobile !== mobile) {
+          setMobile(mobile);
+        }
+      },
+      false
+    );
+  }, [mobile]);
+
```

Comment:
I would definitely recommend adding a cleanup function to this useEffect so we don't just leave that event listener on the window :)

Reply:
Ok, I can add this tomorrow. Thanks.